lib/matplotlib/pyplot.py

```diff
@@ -200,6 +200,126 @@ def uninstall_repl_displayhook():
 
 draw_all = _pylab_helpers.Gcf.draw_all
 
+_ENSURE_AX_DOC = """
+This function has been decorated by pyplot to have
+an implicit reference to the `plt.gca()` passed as the first argument.
+
+The wrapped function can be called as any of ::
+
+   func(*args, **kwargs)
+   func(ax, *args, **kwargs)
+   func(.., ax=ax)
```

Comment:
Maybe replace each of these "func" strings with "{0}" so the actual name can be interpolated.  See below.